lib/http/client.cpp

```diff
@@ -978,6 +978,26 @@ Client::redact_event(const std::string &room_id,
     put<nlohmann::json, mtx::responses::EventId>(api_path, body, std::move(callback));
 }
 
+void
+Client::report_event(const std::string &room_id,
+                     const std::string &event_id,
+                     const std::string &reason,
+                     const int score)
+{
+    const auto api_path = "/client/v3/rooms" + mtx::client::utils::url_encode(room_id) +
+                          "/report/" + mtx::client::utils::url_encode(event_id);
+    nlohmann::json body = nlohmann::json::object();
+    if (!reason.empty())
+        body["reason"] = reason;
+    if (score <= 0 && score <= -100)
```

Comment:
Is there a typo in this condition ? Why check `<= 0` and then `<= -100` ?

Reply:
Oops, thanks for catching that!